Bricks/net/http/test.cc

```diff
@@ -441,10 +441,10 @@ TEST(PosixHTTPServerTest, ChunkedSmoke) {
     const auto offset = body.length() - length;
     for (size_t chunks = length; chunks; chunks = chunks * 2 / 3) {
       std::string chunked_body;
-      for (size_t i = 0; i < chunks; ++i) {
-        const size_t start = offset + length * i / chunks;
-        const size_t end = offset + length * (i + 1) / chunks;
-        chunked_body += current::strings::Printf("%X\r\n", end - start) + body.substr(start, end - start);
+      for (uint64_t i = 0; i < chunks; ++i) {
```

Comment:
OH SHI~!

Then perhaps make the outer look `uint64_t` too?

Sorry, still hungry and still waking up after a lava hike yesterday.